frontend/app_flowy/packages/flowy_infra/lib/theme.dart

```diff
@@ -1,164 +0,0 @@
-import 'package:flowy_infra/color.dart';
-import 'package:flutter/material.dart';
-
-enum ThemeType {
-  light,
-  dark,
-}
-
-//Color Pallettes
-const _black = Color(0xff000000);
-const _grey = Color(0xff808080);
-const _white = Color(0xFFFFFFFF);
-
-class AppTheme {
-  static ThemeType defaultTheme = ThemeType.light;
-
-  bool isDark;
-  late Color surface; //
-  late Color hover;
-  late Color selector;
-  late Color red;
-  late Color yellow;
-  late Color green;
-
-  late Color shader1;
-  late Color shader2;
-  late Color shader3;
-  late Color shader4;
-  late Color shader5;
-  late Color shader6;
-  late Color shader7;
-
-  late Color bg1;
-  late Color bg2;
-  late Color bg3;
-  late Color bg4;
-
-  late Color tint1;
-  late Color tint2;
-  late Color tint3;
-  late Color tint4;
-  late Color tint5;
-  late Color tint6;
-  late Color tint7;
-  late Color tint8;
-  late Color tint9;
-  late Color textColor;
-  late Color iconColor;
-
-  late Color main1;
-  late Color main2;
-
-  late Color shadowColor;
-
-  /// Default constructor
-  AppTheme({this.isDark = false});
-
-  /// fromType factory constructor
-  factory AppTheme.fromType(ThemeType t) {
-    switch (t) {
-      case ThemeType.light:
-        return AppTheme(isDark: false)
-          ..surface = Colors.white
-          ..hover = const Color(0xFFe0f8ff) //
```

Comment:
The color names in AppTheme are specified by the design specs. I think keeping those names is good for development. Or Any suggestions about how to map the color to the design specs.

Reply:
The problem seems like we naming the same thing with different terms, maybe those terms are already covered inside `ThemeData` (or Material Design in general) for around 80% cases (or even higher). Currently we can't add custom color names to it (I have to use `extension`), so I think we should follow Material Design for most of the colors names and having some custom colors for the app.

Reply:
Besides, naming colors with number is very confusing IMO. With `shaders`, `bg` we could use [`elevation`](https://api.flutter.dev/flutter/material/Material/elevation.html) instead (not all of the time), with `tints` we should use the actual color names, with `main1` we have `primary` and `main2` we have `secondary`.

Reply:
> The problem seems like we naming the same thing with different terms, maybe those terms are already covered inside `ThemeData` (or Material Design in general) for around 80% cases (or even higher). Currently we can't add custom color names to it (I have to use `extension`), so I think we should follow Material Design for most of the colors names and having some custom colors for the app.

Agreed! It seems like we need to map the names of the color to the design specs. 
For example:
bg(from design specs)  -> elevation (from the dart code)

So that each developer can find the color when building the UI from design specs.